### PR TITLE
Scaffolder actions

### PR DESCRIPTION
**Pull request by sblausten**
https://app.shortcut.com/larder/story/22851/scaffolder-actions-comprehensive-resource

### CODE REVIEW

**Comment by sblausten** [comment]
Need to add whether each one is available in Roadie

File: content/docs/scaffolder/scaffolder-actions-directory/index.md [hunk 1]
```diff
@@ -5,18 +5,166 @@ description: Directory of available scaffolder actions
...
+| Yeoman               | [run:yeoman](#runyeoman)                  | Action for running a Yeoman generator in Backstage scaffolder.                                                                                                            | [Yeoman Run](https://github.com/backstage/backstage/tree/master/plugins/scaffolder-backend-module-yeoman)                                                                                                                                                                             | namespace*, args, options                                                                               |
+
+## Download content
```
Comment (Xantier) [inline content/docs/scaffolder/scaffolder-actions-directory/index.md:153]: This title doesn't seem like it fits here

File: content/docs/scaffolder/scaffolder-actions-directory/index.md [hunk 2]
```diff
@@ -5,18 +5,166 @@ description: Directory of available scaffolder actions
 ---
 
 
-## Directory of available scaffolder actions
-
-In this page you can find a directory of some available Scaffolder actions and their examples. You can find a full list of available actions to your Roadie instance by visiting the following page from within Roadie:
-
-`https://<tenant-name>.roadie.so/templates/actions`
+# Directory of available scaffolder actions
```
Comment (punkle) [inline content/docs/scaffolder/scaffolder-actions-directory/index.md:8]: There is a difference between the actions available in Roadie and the actions available for backstage. For the plugins we have two difernent pages for this.. the open source plugins directory for backstage is under resources off the main page.

By documenting all of the actions here I think implies that these actions are available on roadie.

Comment (sblausten) [reply content/docs/scaffolder/scaffolder-actions-directory/index.md:8]: Yea I'll change "available". I'm not sure splitting them up is very helpful though from a SEO standpoint. I prefer to mark which actions are available in Roadie as mentioned in my comment. I think we will be revising this content into more digestible formats anyway.

File: content/docs/scaffolder/scaffolder-actions-directory/index.md (hunk 1 above)
Comment (sblausten) [reply content/docs/scaffolder/scaffolder-actions-directory/index.md:153]: What category would you call these then?

Comment (Xantier) [reply content/docs/scaffolder/scaffolder-actions-directory/index.md:153]: It's not the category. There is a full title missing. Now it's abruptly changing into a different list after the table.

File: content/docs/scaffolder/scaffolder-actions-directory/index.md (hunk 2 above)
Comment (punkle) [reply content/docs/scaffolder/scaffolder-actions-directory/index.md:8]: Perfect 👌 , so long as its clear. btw.. the roadie supported actions directory is already here https://roadie.io/docs/scaffolder/scaffolder-actions-directory/

So I guess you are creating duplication by creating this page?

File: content/docs/scaffolder/scaffolder-actions-directory/index.md (hunk 1 above)
Comment (sblausten) [reply content/docs/scaffolder/scaffolder-actions-directory/index.md]: Is that better now?

File: content/docs/scaffolder/scaffolder-actions-directory/index.md (hunk 2 above)
Comment (sblausten) [reply content/docs/scaffolder/scaffolder-actions-directory/index.md]: No you can see in the changelog that this is an update to the original page

File: content/docs/scaffolder/scaffolder-actions-directory/index.md (hunk 1 above)
Comment (Xantier) [reply content/docs/scaffolder/scaffolder-actions-directory/index.md]: yup, looks good :+1:

**Pull request merged**